_layout/head.html

```diff
@@ -28,7 +28,11 @@
 
 {{ispage /blog/*}}
 <div class="container blog-title">
-	{{isdef title}}<h1>{{fill title}}</h1>{{end}}
+  {{isdef title}}<h1>{{fill title}}
+    <a type="application/rss+xml" href="https://julialang.org/feed.xml">
+      <i style="color:#e2802f" class="fa fa-rss-square"></i>
+    </a>
+  </h1>{{end}}
```

Comment:
This will show  on *every* blog post, if that's fine then :+1:

Reply:
I think it should occur wherever possible, I thought this would be better than keeping it in navbar.